tests/system/webdriver/tests/users/UserNotesManager0001Test.php

```diff
@@ -257,7 +257,7 @@ public function setFilter_TestOrdering_ShouldOrderNotes()
 		$user2Note = 'This is another user note with custom fields.';
 		$this->userNotesManagerPage->addUserNotes($user2NotesName, $userName2, array('Category' => $category, 'Status' => $user2Status, 'Review Date' => $user2ReviewTime, 'Note' => $user2Note));
 
-		$orderings = array('User', 'Subject', 'Category', 'Status', 'Review date', 'ID');
+		$orderings = array('User', 'Subject', 'Category', 'Status', 'Review Date', 'ID');
```

Comment:
How is this related to 
https://github.com/joomla-projects/GSOC-Webdriver_system_tests_for_CMS/pull/119/files#diff-bcec3fce941e769300246a4409db9322R262

We are having a loop for $actualRowNumbers, I can name it what ever I want, we don't need to mix both of them..I can say 
```
foreach ($actualRowNumbers as $something => $orderingRowNumbers)
```